daemon/policy.go

```diff
@@ -98,6 +98,43 @@ func (d *Daemon) TriggerPolicyUpdates(added []policy.NumericIdentity) {
 	d.endpointsMU.RUnlock()
 }
 
+func (d *Daemon) UpdatePolicyEnforcement(e *endpoint.Endpoint) bool {
```

Comment:
Can you add a description to the function? I don't know what's the meaning of the bool returned